### PR TITLE
feat(issue-30): show pull request a name on toast message

### PR DESCRIPTION
**Pull request by jskurt**
## Issue Link

[issue-30](https://github.com/kafelix496/coding-for-fun/issues/30)

## Notes

- I wanna more pretty toast like that
[SUCCESS] Repository: xxx
PR: xxx
COMMENT: xxx
but I can't. because I don't know how to use 'enter?' like '\n'.
How can we use it? Do you know that? Our friend ChatGPT doesn't know also.

- You didn't approve this PR(#35) but I can merge. I mean merge button is active. Is it right? maybe something problem.

### CODE REVIEW

**Comment by jskurt** [comment]
image
I modified. Do you like it? 😎

**Comment by jskurt** [comment]
image
Fianl agreement 💯

File: src/modules/github/components/pull-review-form/pull-review-form.tsx
Comment (kafelix496) [inline src/modules/github/components/pull-review-form/pull-review-form.tsx:86]: Please use the internalization feature.

File: src/elements/toast/toast.tsx
Comment (kafelix496) [inline src/elements/toast/toast.tsx:121]: I don't want to add this style here.
Instead of adding another style, please consider passing JSX Element instead of a string when you are calling `pushToast` function.

here's an example
```typescript
pushToast({
  title: 'hello'
  description: (
    <div>
      <div>world1</div>
      <div>world2</div>
    </div>
  )
})
```

**Comment by jskurt** [comment]
image
image

**Comment by jskurt** [comment]
I'm passed, but I have a problem that approve button always fail from repository test-repo-2, test-repo-3. Is that because it's my repository and I only need to approve it once?

**Comment by kafelix496** [comment]
> I'm passed, but I have a problem that approve button always fail from repository test-repo-2, test-repo-3. Is that because it's my repository and I only need to approve it once?

It's because the pull request is created by you and you can't approve it yourself. It's expected behavoir.

**Review by kafelix496 — APPROVED** [review]

**Pull request merged**
